boards/arm/ruuvi_ruuvitag/doc/nrf52_ruuvi.rst

```diff
@@ -0,0 +1,182 @@
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| SPI       | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+| Humidity, | on-board   | bme280               |
+| Temp & Air|            |                      |
+| Pressure  |            |                      |
++-----------+------------+----------------------+
+| Acc       | on-board   | lis2dh12             |
++-----------+------------+----------------------+
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED0 (red) = P0.17
+* LED1 (green) = P0.19
+
+Push buttons
+------------
+
+* BUTTON0 = SW1 = P0.13
+
+Pin descriptions
+----------------
+
+.. figure:: img/pinout.jpg
+     :width: 442px
+     :align: center
+     :alt: RUUVI Pinout
+
+* 2 = P0.29 = SPI_SCK
+* 3 = P0.28 = SPI_MISO
+* 10 = P0.04 = GPIO (can be used as a GPIO / ADC pin)
+* 11 = P0.05 = GPIO (can be used as a GPIO / ADC pin)
+* 12 = P0.25 = SPI_MOSI
+* 13 = P0.19 = LED2 (green) / GPIO (can be used as a GPIO pin but the LED will blink)
+* 14 = P0.17 = LED1 (red) / GPIO (can be used as a GPIO pin but the LED will blink)
+* 15 = P0.13 = Button / GPIO (can be used as a GPIO pin)
+* 16 = GND (Battery's negative contact)
+* 17 = Battery's positive contact
+* 18 = Battery's positive contact
+* 19 = SWDIO
+* 20 = SWDCLK
+* 21 = P0.18 = SWO / GPIO (can be used as a GPIO pin)
+* 22 = P0.21 = Reset / GPIO (can be used as a GPIO pin if no need to reset the device)
+* 23 = GND (Battery's negative contact)
+* 24 = P0.31 = GPIO (can be used as a GPIO / ADC pin)
+* 25 = P0.30 = GPIO (can be used as a GPIO / ADC pin)
+
+GPIO = General Purpose Input Output pin
+
+P1 = Standard 10-pin ARM Cortex debug connector (on RuuviTag Rev.B1-B5)
+
+* 1 = VDD
+* 2 = SWDIO
+* 3 = GND (Battery's negative contact)
+* 4 = SWDCLK
+* 5 = GND (Battery's negative contact)
+* 6 = SWO
+* 7 = No Connect
+* 8 = No Connect
+* 9 = GND (Battery's negative contact)
+* 10 = Reset
+
+P1 = TC2030 TagConnect (on RuuviTag Rev.B6)
+
+* 1 = Battery's positive contact
+* 2 = SWDIO
+* 3 = Reset
+* 4 = SWDCLK
+* 5 = GND (Battery's negative contact)
+* 6 = SWO
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Flashing Zephyr onto a RuuviTag requires an external Ruuvi DEVKIT. 
```

Comment:
Maybe link to this devkit and also mention if this can be achieved as well with a Nordic devkit?